format/spec.md

```diff
@@ -630,6 +630,21 @@ When expiring snapshots, retention policies in table and snapshot references are
     2. The snapshot is not one of the first `min-snapshots-to-keep` in the branch (including the branch's referenced snapshot)
 5. Expire any snapshot not in the set of snapshots to retain.
 
+#### Statistics file
```

Comment:
I think this should start with some context about stats files. I think that we require them to be valid Puffin files and explain how they're used. They are also informational. The paragraph below the table is good and can be moved up here to clarify that.

Reply:
good idea, added a word of intro, please take another look